packages/version-bump/src/tests/utils.ts

```diff
@@ -48,187 +49,198 @@ export const checkForDuplicateCases = (testCases: Array<Record<string, unknown>>
 
 export const mainNormalTestCases: Array<MainTestCase> = [
 	{
-		type: 'pre_release',
+		type: 'major',
 		inputVer: '4.10.9.rc.011',
-		outputVer: '4.10.9.beta',
+		// major is incremented and minor and patch are set to 0
+		outputVer: '5.0.0.rc',
 	},
 	{
-		type: 'pre_release',
+		type: 'major',
 		inputVer: '4.10.9.rc',
-		outputVer: '4.10.9.beta',
+		outputVer: '5.0.0.rc',
 	},
 	{
-		type: 'pre_release',
+		type: 'major',
 		inputVer: '4.10.9.beta',
-		outputVer: '4.10.9.beta',
+		outputVer: '5.0.0.beta',
 	},
 	{
-		type: 'pre_release',
-		inputVer: '0.0.0',
-		outputVer: '0.0.0.beta',
+		type: 'major',
+		inputVer: '4.8.0.beta',
+		// no change to patch when it's already 0
+		outputVer: '5.0.0.beta',
 	},
 	{
-		type: 'decaf',
-		inputVer: '4.10.9.rc.011',
-		outputVer: '4.10.9.decaf',
+		type: 'major',
+		inputVer: '4.0.8.beta',
+		// no change to minor when it's already 0
+		outputVer: '5.0.0.beta',
 	},
 	{
-		type: 'decaf',
-		inputVer: '4.10.9.rc',
-		outputVer: '4.10.9.decaf',
+		type: 'major',
+		inputVer: '4.0.0.beta',
+		// no change to minor and patch when already 0
+		outputVer: '5.0.0.beta',
 	},
 	{
-		type: 'decaf',
-		inputVer: '4.10.9.beta',
-		outputVer: '4.10.9.decaf',
+		type: 'major',
+		inputVer: '0.0.0',
+		outputVer: '1.0.0.rc',
```

Comment:
hmmm... since everything else is working more explicitly now, I wonder if we should not append `rc` by default, and instead force setting the release type to also be an explicit action so that nothing happens via "magic"... thoughts?

Reply:
Does that mean if no release type is passed, nothing will be appended or existing will be retained?

Reply:
imho unless you are setting the release type, then the release type should not change, ie NO magic!
that said, it's totally fine to have `rc` as the default value when setting the release type.

Reply:
you mentioned in chat:

> I am thinking to add an explicit option for setting the releaseType along with the version bump

which is fine since that would be an explicitly requested action...

ie: "do A + B" resulting in "A + B" happening is totally fine

but "do A" resulting in "A + B" is not

Reply:
Thanks